pipelines/SARS-CoV-2-pipelines/run_python_scripts/rules/helpers.py

```diff
@@ -0,0 +1,99 @@
+import math
+import os
+from collections import defaultdict
+
+letters = "ACGT"
+l2n = {letter: num for num, letter in enumerate(letters)}
+
+def load_dict(file_path, reference):

```

Comment:
isn't this same as in compare_mutations.py?

Reply:
it is. I'll import it to compare_mutations from helpers. same in count_observed_counts.